### PR TITLE
Sonnet V1.2.9

### PR DESCRIPTION
**Pull request by ultrabear**
A small QOL update with the backend changes needed to implement an faq module

### CODE REVIEW

File: cmds/cmd_version.py [hunk 1]
```diff
@@ -80,7 +94,12 @@ async def print_version_info(message: discord.Message, args: List[str], client:
 
     fmt.write(f"\nBot Uptime: {getdelta(bot_start_time)}\n```")
 
-    await message.channel.send(fmt.getvalue())
+    content = fmt.getvalue()
+
+    if len(content) <= 2000:
+        await message.channel.send(content)
+    else:
+        await message.channel.send("ERROR: Exeeded discord message length limits, tell a developer to stop being lazy about rendering this")
```
Comment (Saragon4005) [inline cmds/cmd_version.py:102]: if I tell you here it kills the fun right?

File: libs/lib_encryption_wrapper.py [hunk 2]
```diff
@@ -1,12 +1,14 @@
 # Cryptographically sound AES-CTR-HMAC wrapper for fileio
 # Ultrabear 2021
 
+# This module sucks, python was not meant to do disk handling
```
Comment (Saragon4005) [inline libs/lib_encryption_wrapper.py:4]: character development

File: cmds/cmd_version.py (hunk 1 above)
Comment (ultrabear) [reply cmds/cmd_version.py:102]: gonna wait till it actually becomes a problem to deal with it

File: libs/lib_loaders.py [hunk 3]
```diff
@@ -31,7 +31,7 @@
 
 class DotHeaders:
 
-    version = "1.2.8-DEV.1"
+    version = "1.2.9-DEV.0"
```
Comment (Saragon4005) [inline libs/lib_loaders.py:34]: uh why change this? here did literally anything else change in net?

Comment (ultrabear) [reply libs/lib_loaders.py:34]: dotheaders manages gcc compiling it
it wont recompile and use the new binary unless the version changes

**Review by Saragon4005 — APPROVED** [review]

**Comment by ultrabear** [comment]
# Sonnet V1.2.9
## Frontend:
- Add(remove-regexblacklist): Remove a regex by its index
- Change(search-infractions): Only specifying a uid will default to user infractions
- Change(help): Typing `help 2` etc for a second help page will prompt the user the correct syntax
## Under the Hood:
- Add(clang-format): C formatter
- Change(cloader): Make cloader use long in place of int for arbitrary temp numbers
- Add(slots): Add dunder slots to many classes, lowering class overhead
- Add(format_duration): End user duration formatting for arbitrary seconds (used in mute)
- Change(lib_db_handler): Libraries are now more memory efficient by replacing list comprehensions with generators
- Add(db_hlapi.list_enum): list_enum call to list all primary keys in an enum
- Add(LeXdPyK): 1.2.6: Previously mentioned dunder slots and encoding specifiers
## Bugs:
- Patch(encoding): Make all file opens specify utf8 encoding

**Pull request merged**